src/code/engine_library/calculatePopulationPKParameter.m

```diff
@@ -12,29 +12,37 @@ function calculatePopulationPKParameter(WSettings,PopRunSet)
 
 % Open Systems Pharmacology Suite;  http://open-systems-pharmacology.org
 
+try
+    writeToReportLog('INFO',sprintf('Calculate PK Parameter of %s',PopRunSet.name),false);
+    
+    
+    % load population
+    load(fullfile('tmp',PopRunSet.name,'pop.mat'),'parPaths','parValues');
+    
+    % reduce to demographic
+    [jj,ix] = ismember({'IndividualId','Organism|Weight','Organism|Height','Organism|BMI'},parPaths); %#ok<NODEF>
+    parPaths = parPaths(ix(jj));
```

Comment:
later on we should define constants for those parameters ('Organism|Weight' etc.)